Add a webpage that walks through the new backup and restore examples for a local Vitess environment. (#1861)
vitessio/website: content/en/docs/22.0/user-guides/configuration-advanced/resharding.md

## Next Steps

Congratulations! You have successfully resharded your customer keyspace into two shards. Now, let's learn how to take backups of your Vitess cluster.

> * For Local Environment: If you are using a local machine, you can refer to the section on [Backups and Restore for Local Environment](../../operating-vitess/backup-and-restore/backup_and_restore_local) to perform backups and restorations.
> 
> * For Kubernetes Environment: If you are using Kubernetes, you can follow the instructions on [how to schedule backups](../../operating-vitess/backup-and-restore/scheduled-backups) to automate your backup processes.

Reviewer: ```suggestion
* For local environments: You can refer to the section on [Backups and Restore for Local Environment](../../operating-vitess/backup-and-restore/backup_and_restore_local) to backup and restore your cluster.

* For Kubernetes environments: You can follow the instructions on [how to schedule backups](../../operating-vitess/backup-and-restore/scheduled-backups) to automate your backup process.
```
